RunCat365/ContextMenuManager.cs

```diff
@@ -144,10 +147,38 @@ Action onExit
 
             SetIcons(getSystemTheme(), getManualTheme(), getRunner());
 
-            notifyIcon.Text = "-";
-            notifyIcon.Icon = icons[0];
-            notifyIcon.Visible = true;
-            notifyIcon.ContextMenuStrip = contextMenuStrip;
+            notifyIcon = CreateInitialNotifyIcon();
+
+            SystemEvents.PowerModeChanged += (_, e) =>
+            {
+                if (e.Mode == PowerModes.Resume)
+                {
+                    RecreateNotifyIcon();
+                }
+            };
+        }
+
+        private NotifyIcon CreateInitialNotifyIcon()
+        {
+            return  new()
+            {
+                Text = iconText,
+                Icon = icons[0],
+                Visible = true,
+                ContextMenuStrip = contextMenuStrip
+            };
+        }
+
+        private void RecreateNotifyIcon()
+        {
+            try
+            {
+                notifyIcon.Dispose();
+            } catch { }
+
+            notifyIcon = CreateInitialNotifyIcon();
```

Comment:
If you are going to reuse it, the function name `CreateInitialNotifyIcon` is inappropriate.